Gemfile

```diff
@@ -19,7 +19,7 @@ if defined? JRUBY_VERSION
   gem 'jdbc-mysql'
   gem 'traject-marc4j_reader'
 #  gem 'jruby-launcher', '~>1.1'
-#  gem 'naconormalizer', path: '../naconormalizer'
+  gem 'naconormalizer'
```

Comment:
The checked-out naconormalizer at `/htsolr/catalog/bin/naconormalizer` seems to be the latest released version from rubygems. Is there any reason to prefer what's on github to rubygems to a checked-out version?

Reply:
I'm confused as to whether anything needs to change here now or not.

Reply:
I'm doing a fresh pull and test against with the gem and will let you know how it goes. 99% sure it'll be fine.